### PR TITLE
Add flavour GCP volume size

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: Tzvika Avni <tavni@redhat.com>

SDA-6561

@etabak @zgalor @oriAdler

### CODE REVIEW

File: model/clusters_mgmt/v1/gcp_flavour_type.model [hunk 1]
```diff
@@ -25,5 +25,13 @@ struct GCPFlavour {
 	ComputeInstanceType String
 
 	// GCP default instance type for the infra volume.
-	InfraInstanceType String
+
```
Comment (zgalor) [inline model/clusters_mgmt/v1/gcp_flavour_type.model:28]: Did you intend to remove the InfraInstanceType?

Comment (zvikorn) [reply model/clusters_mgmt/v1/gcp_flavour_type.model:28]: no thanks!

**Comment by zgalor** [comment]
LGTM

**Pull request merged**